code/datums/mutations/passive.dm

```diff
@@ -11,3 +11,26 @@
 /datum/mutation/human/biotechcompat/on_losing(mob/living/carbon/human/owner)
 	owner.adjust_skillchip_complexity_modifier(-1)
 	return ..()
+
+/datum/mutation/human/good_looking
```

Comment:
just fyi, this means anyone can find this mutation naturally. if you don't want that, take a look at how the holy mutations (honorbound, burdened) work to not be in the pool

Reply:
yeah looking good is reserved for Habsburgs